packages/salesforcedx-vscode-core/src/commands/forceSourceStatus.ts

```diff
@@ -18,23 +18,41 @@ import {
 } from './commands';
 
 class ForceSourceStatusExecutor extends SfdxCommandletExecutor<{}> {
+  private flag: string | undefined;
```

Comment:
I think this should be a String enum given that we have only two choices: --local or --remote. This helps prevent failures in the future by enumerating the choice of everything that is possible.

Either use that or the "String Literal Types". See https://www.typescriptlang.org/docs/handbook/advanced-types.html